src/common/filter/Expressions.cpp

```diff
@@ -241,8 +241,7 @@ Status VariablePropertyExpression::prepare() {
 
 
 OptVariantType EdgeTypeExpression::eval(Getters &getters) const {
-    UNUSED(getters);
-    return *alias_;
+    return getters.getAliasProp(*alias_, *prop_);
```

Comment:
Be careful memory leaks. memory leaks occur when getters.getAliasProp == nullptr . right?

Reply:
`getters ` needs to guarantee that getAliasProp is not null